builtin/providers/aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -219,6 +219,21 @@ func testAccCheckAWSElasticacheClusterExists(n string, v *elasticache.CacheClust
 	}
 }
 
+var testAccAWSElasticacheClusterConfigBasic = fmt.Sprintf(`
+provider "aws" {
+	region = "us-east-1"
+}
+
+resource "aws_elasticache_cluster" "bar" {
+    cluster_id = "tf-%s"
+    engine = "memcached"
+    node_type = "cache.m1.small"
+    num_cache_nodes = 1
+    port = 11211
+    parameter_group_name = "default.memcached1.4"
+}
+`, acctest.RandString(10))
```

Comment:
nit: if we ever re-use `testAccAWSElasticacheClusterConfigBasic`, it will have the same `cluster_id` as the `accent.RandString` is only ran on initialization. For future the configs should be methods that receive the random thing and insert it, similar to https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_elastic_beanstalk_environment_test.go#L88